evap/static/ts/src/infobox.ts

```diff
@@ -0,0 +1,40 @@
+declare const bootstrap: typeof import("bootstrap");
+
+import { selectOrError } from "./utils.js";
+
+const OPEN_CLOSE_TIMEOUT = 2000;
+
+export class InfoboxLogic {
+    private readonly infobox: HTMLDivElement;
+    private readonly closeButton: HTMLButtonElement;
+    private readonly storageKey: string;
+
+    constructor(infobox_id: string) {
+        this.infobox = selectOrError("#" + infobox_id);
+        this.closeButton = selectOrError("#" + infobox_id + " .callout-infobox-close");
```

Comment:
Shouldn't this be something similar to:
```suggestion
        this.closeButton = selectOrError(".callout-infobox-close", this.infobox);
```